pkgs/by-name/sp/spades/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  zlib,
+  bzip2,
+  cmake,
+  python3,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "spades";
+  version = "3.15.5";
+
+  src = fetchurl {
+    url = "https://github.com/ablab/spades/releases/download/v${finalAttrs.version}/SPAdes-${finalAttrs.version}.tar.gz";
+    hash = "sha256-FVw2QNVx8uexmgUDHR/Q0ZvYLfeF04hw+5O9JBsSu/o=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    zlib
+    bzip2
+    python3
+  ];
+
+  doCheck = true;
+
+  sourceRoot = "SPAdes-${finalAttrs.version}/src";
+
+  env.CXXFLAGS = toString [
```

Comment:
`env` is configure phase. Follow phases order:

https://nixos.org/manual/nixpkgs/stable/#sec-stdenv-phases

(Move it to before build phase)

Reply:
i can't find env in the docs :skull:
nix* docs as good as always

Reply:
`env` being environment variable. `CXXFLAGS` being configure phase. This is part of 'configuration' of build.
Just move it to configure phase.